analytics/views.py

```diff
@@ -1092,12 +1116,25 @@ def support(request: HttpRequest) -> HttpResponse:
 
         context["realms"] = realms
 
+        cnfs = []  # type: List[Dict[str, Any]]
```

Comment:
better not to use an abbreviation for this, I tihnk

Reply:
Done.